src/pages/Main/Main.js

```diff
@@ -1,7 +1,23 @@
+import { useState } from 'react';
+import LoginPage from '../../components/Common/LoginAndJoin/LoginPage/LoginPage';
 import './Main.scss';
 
 const Main = () => {
-  return <main>this is main</main>;
+  const [login, setLogin] = useState(false);
+
+  return (
+    <main>
+      <button
+        onClick={() => {
+          console.log(login);
+          login === false ? setLogin(true) : setLogin(false);
+        }}
+      >
+        Login
+      </button>
+      {login === true ? <LoginPage login={login} /> : null}
```

Comment:
논리곱 단축평가(`&&`) 이용해서 나타낼 수 있습니다.

```suggestion
      {login && <LoginPage login={login} />}
```